src/types/geometrycollection.rs

```diff
@@ -21,16 +21,17 @@ use crate::{FromTokens, Wkt, WktNum};
 use std::fmt;
 use std::str::FromStr;
 
+/// A parsed GeometryCollection.
```

Comment:
nitpicking: What's the added utility of the word `parsed` here? You could build this declaratively too right?

Reply:
Yes, you could. 

The point of `parsed` here is that the overhead to arrive at a `GeometryCollection` from an arbitrary string has already been done. So each coordinate access from a `GeometryCollection` is free; you don't have to parse a WKT string again.

I think this is especially relevant for the type called `Wkt`. It wouldn't be that surprising if the `Wkt` struct held a _string_ or _string-like_ object, i.e. _before_ parsing to a geometry. Whereas in actuality the `Wkt` is the _result_ of parsing a WKT string. I can take out the word `parsed` but I thought maybe it helped to make the sentiment clearer.